pyproject.toml

```diff
@@ -7,19 +7,14 @@ package-mode = false
 [tool.poetry.dependencies]
 python = "^3.10"
 ops = "^2.17.1"
-cryptography = "^44.0.0"
 boto3 = "^1.35.87"
 pgconnstr = "^1.0.1"
 requests = "^2.32.3"
 tenacity = "^9.0.0"
 psycopg2 = "^2.9.10"
-cosl = "^0.0.50"
 pydantic = "^1.10.19"
-poetry-core = "^2.0.0"
-pyOpenSSL = "^24.3.0"
 jinja2 = "^3.1.4"
 pysyncobj = "^0.3.13"
-psutil = "^6.1.1"
```

Comment:
Psutil will be needed by https://github.com/canonical/postgresql-operator/pull/663 and https://github.com/canonical/postgresql-operator/pull/611